### PR TITLE
status: 2023q1: bsd-user: correction, suggestions

### PR DESCRIPTION
**Pull request by grahamperrin**
@bsdimp

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/bsd-user.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ link:https://gitlab.com/qemu-project/qemu[QEMU Project's gitlab mirror] URL: lin
 
 Contact: Warner Losh <imp@FreeBSD.org>
 
-In this quarter, Warner upstreamed patche sets in the qemu-project repo (with a third pending).
+In this quarter, Warner upstreamed two patch sets in the qemu-project repo (with a third pending).
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/bsd-user.adoc:10]: For a third thing to be pending, I assume that the two things preceding were _patch sets_.

True, @bsdimp?

**Comment by bsdimp** [comment]
These are all correct

**Comment by grahamperrin** [comment]
@bsdimp thanks. 

@lsalvadore I didn't await additional approval from `status@` because the few changes here were simple and non-contentious. Hope that's OK.

**Comment by lsalvadore** [comment]
@grahamperrin, sure no problem. Thanks for your work and feel free to proceed again without any status approval in similar situations.

**Pull request closed without merging**